src/ontology/hp-edit.owl

```diff
@@ -161749,7 +161749,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0031574> <http://purl.obolibrary.o
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:2503273") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0031575> "As in the Number 1 and Number 2 clefts, this cleft extends through the lip in the region of the typical cleft lip; however, it does not extend through the base. The cleft continues superiorly to involve the inner canthus and lower eyelid medial to the inferior lacrimal punctum, thereby disruptiong the nasolacrimal system. Microphthalmia may be present. The alveolar cleft is between the lateral incisor and the canine. Absent septation between the nasal cavity and maxillary antrum, together with the distortion of the frontal process of the maxilla and lacrimal fossa, produces direct communication between the orbit, maxillary sinus, and nose. There is hypoplasia of the soft tissue margins of the cleft in the vertical dimension. This produces extreme soft tissue deficiency between the alar base and the cleft of the medial aspect of the lower eyelid. The inferior lacrimal punctum is evident at the lateral margin of the lower eyelid cleft. The lacrimal drainage system ends as an opening directly onto the cheek without communication into the nasal cavity. The globe is normal in size, but it is displaced inferiorly and laterally. The nasal septum shows the characteristic distortion seen in typical cleft lip and palate. There is absence of septation between the nasal cavity on the cleft side and the maxilla. The maxilla is hypoplastic in three dimensions, with a marked reduction in pneumatization. Superior extension of the skeletal clefting into the medial portion of the orbital floor and into the inferior orbital rim in the region of the frontal process of the maxilla allows direct communication between the orbit above and the nasomaxillary region below. There is mild narrowing of the ethmoid sinus and of the body of the sphenoid on the cleft side. The pterygoid process appears anatomically normal, but less displaced from the midline compared with that of the noncleft side. Both the orbit and the floor of the anterior cranial fossa are inferiorly displaced.")
```

Comment:
It was hard to find the differences in this, but the one correction I found is incorrect. It should be the original 'disrupting